kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-explore-topic.handlebars

```diff
@@ -3,17 +3,17 @@
   <div class="content-explore-topic">
     <div class="row">
       <div class="col-xs-12">
-          <h4><small>{{_ "Based on your interest in" }} {{ interest_topic.title }}:</small></h4>
+          <p class="up10">{{_ "Based on your interest in" }} <b>{{ interest_topic.title }}</b><span class="sr-only">{{_ ", we recommend you" }}</span>:</p>
       </div>
     </div>
     <div class="row">
       <div class="col-xs-5">
-          <h3>{{ suggested_topic.title }}:</h3>
+          <h2 class="h3">{{ suggested_topic.title }}:</h2>
       </div>
       {{#if suggested_topic.description }}
       <div class="col-xs-7">
-        <h4>{{_ "Description" }}:</h4>
-        <p>{{ truncate suggested_topic.description 70 }}</p>
+        <p class="h4">{{_ "Description" }}:</p>
+        <p>{{ truncate suggested_topic.description 100 }}</p>
```

Comment:
Since screen reader users can easily skip any element they do not have the desire to hear, I chose to make the full description available. This aligns with the general policy to make the same content available for non sighted users as for the sighted ones, so they can freely choose what to use and what to ignore, just as the rest of us.